caribou/monitors/deployment_manager.py

```diff
@@ -46,17 +46,20 @@
 
 
 class DeploymentManager(Monitor):
-    def __init__(self, deployment_metrics_calculator_type: str = "simple") -> None:
+    def __init__(self, deployment_metrics_calculator_type: str = "simple", lambda_timeout: bool = False) -> None:
         super().__init__()
         self.workflow_collector = WorkflowCollector()
         self._deployment_metrics_calculator_type: str = deployment_metrics_calculator_type
+        self._lambda_timeout: bool = lambda_timeout
 
     def check(self) -> None:
+        print(f"Running Deployment Manager: Manage Deployments")
```

Comment:
Did we not use logging instead?

Reply:
Seem like print does in fact show up in CloudWatch. But you are right where its probably better to use info logging. I have applied this change.